sql/12.16/extra/int4.sql

```diff
@@ -369,6 +369,28 @@ INSERT INTO INT4_TMP VALUES (2147483647::int4, 2147483646::int4);
 --Testcase 134:
 SELECT lcm(f1, f2) FROM INT4_TMP; -- overflow
 
+--Testcase 135:
+DELETE FROM INT4_TMP;
+--Testcase 136:
+ALTER FOREIGN TABLE INT4_TMP ALTER COLUMN f1 TYPE int8;
+--Testcase 137:
+INSERT INTO INT4_TMP VALUES (x'7FFFFFFF'::int8 + 1, 0);
+--Testcase 138:
+ALTER FOREIGN TABLE INT4_TMP ALTER COLUMN f1 TYPE int4;
+--Testcase 139:
+SELECT * FROM INT4_TMP; -- overflow
+
+--Testcase 140:
+DELETE FROM INT4_TMP;
```

Comment:
Could you add test code for int2 overflow?

Reply:
Added in https://github.com/pgspider/sqlite_fdw/commit/641c539615b36ff203199b0c908524cf069ba358. Will rewritten to PostgreSQL core functions in next commit.